defmt/src/impls/core_/net.rs

```diff
@@ -0,0 +1,69 @@
+use core::net;
+
+use super::*;
+
+impl Format for net::AddrParseError {
+    fn format(&self, fmt: Formatter) {
+        crate::write!(fmt, "AddrParseError(_)");
+    }
+}
+
+impl Format for net::Ipv4Addr {
+    fn format(&self, fmt: Formatter) {
+        crate::write!(
+            fmt,
+            "{}.{}.{}.{}",
+            self.octets()[0],
+            self.octets()[1],
+            self.octets()[2],
+            self.octets()[3]
+        );
```

Comment:
You _could_ also do the following. You don't have to; I just find it pretty.

```suggestion
        let [a, b, c, d] = ip.octets();
        crate::write!(fmt, "{}.{}.{}.{}", a, b, c, d);
```

With `println!` you could even do following, but unfortunately we didn't implement that (yet):

```rust
let [a, b, c, d] = ip.octets();
println!("{a}.{b}.{c}.{d}");
```

Reply:
Done!